tests/bootstrap.php

```diff
@@ -8,10 +8,10 @@
 	die( 'You need to install this package with Composer before you can run the tests' );
 }
 
-$classLoader = require_once( __DIR__ . '/../vendor/autoload.php' );
+$classLoader = require_once ( __DIR__ . '/../vendor/autoload.php' );
```

Comment:
The added space is very odd. This should either be `function( $string )` or `keyword $string`.